test/ussd_higherhealth_healthcheck.test.js

```diff
@@ -373,6 +376,9 @@ describe("ussd_higherhealth_healthcheck app", function () {
         it("should display an error on invalid input", function () {
             return tester
                 .setup.user.state("state_age")
+                .setup.user.answers({
+                    state_province: 'ZA-GT',
+                })
```

Comment:
I'm not sure why we're setting up `state_province` here? We haven't made any changes to `state_age` for it to require the province. I think we only need `state_province` on the `should go to state_university` test, since then it's going to `state_university` where it needs the province.